settings.gradle

```diff
@@ -2,6 +2,7 @@ include 'api'
 include 'app'
 include 'protocol'
 include 'client'
+include 'testing/container'
```

Comment:
module can be just `testing`

Reply:
It is including the directory beneath ? it is from ` testing/container/src/...` to `testing/src/...` isn't it ?